Exiled.Events/Patches/Events/Scp049/StartingAndFinishingRecall.cs

```diff
@@ -60,23 +62,28 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 new(OpCodes.Brfalse_S, ret),
             });
 
-            offset = -2;
-            index = newInstructions.FindLastIndex(instruction => instruction.opcode == OpCodes.Ldc_I4_S) + offset;
+            offset = 0;
```

Comment:
If the required offset is 0 then `offset` can be set to a `const` of -4 and this can be removed.